packages/cfd/src/Containers/__tests__/cfd-dashboard-container.spec.js

```diff
@@ -0,0 +1,107 @@
+            'href',
+            'https://www.metatrader5.com/en/terminal/help/start_advanced/install_linux'
+        );
+        expect(screen.getByText(/IcInstallationGoogle/i).closest('a')).toHaveAttribute(
+            'href',
+            'https://download.mql5.com/cdn/mobile/mt5/android?server=Deriv-Demo,Deriv-Server'
+        );
+        expect(screen.getByText(/IcInstallationApple/i).closest('a')).toHaveAttribute(
+            'href',
+            'https://download.mql5.com/cdn/mobile/mt5/ios?server=Deriv-Demo,Deriv-Server'
+        );
+    });
+
+    it('should render correct text according to the platform (DerivX)', () => {
+        render(<CFDDashboardContainer {...mock_props} platform='dxtrade' is_dark_mode_on='true' />);
+        expect(screen.getByText(/Run Deriv X on your browser or download the mobile app/i)).toBeInTheDocument();
+        expect(screen.getByText(/Web terminal/i)).toBeInTheDocument();
+        expect(screen.getByTestId(/dt_dxtrade_desktop_download/i)).toBeInTheDocument();
+    });
+    it('should render the correct icons for the Deriv X platform', () => {
+        render(<CFDDashboardContainer {...mock_props} platform='dxtrade' />);
+        expect(screen.getByText(/IcDxtradeDeviceDesktop/i)).toBeInTheDocument();
+        expect(screen.getByText(/IcDxtradeDeviceLaptop/i)).toBeInTheDocument();
+        expect(screen.getByText(/IcBrandDxtrade/i)).toBeInTheDocument();
+        expect(screen.getByText(/IcDxtradeDeviceTablet/i)).toBeInTheDocument();
+        expect(screen.getByText(/IcDxtradeDevicePhone/i)).toBeInTheDocument();
+        expect(screen.getByText(/IcInstallationGoogle/i)).toBeInTheDocument();
+        expect(screen.getByText(/IcInstallationApple/i)).toBeInTheDocument();
+    });
+    it('should render the correct icons if dark mode is on for DerivX', () => {
+        render(<CFDDashboardContainer {...mock_props} platform='dxtrade' is_dark_mode_on='true' />);
+        expect(screen.getByText(/IcDxtradeDeviceDesktopLight/i)).toBeInTheDocument();
+        expect(screen.getByText(/IcDxtradeDeviceLaptopLight/i)).toBeInTheDocument();
+        expect(screen.getByText(/IcDxtradeDeviceTabletLight/i)).toBeInTheDocument();
+        expect(screen.getByText(/IcDxtradeDevicePhoneLight/i)).toBeInTheDocument();
+    });
+
+    it('should download/redirect the correct file for DerivX', () => {
+        render(<CFDDashboardContainer {...mock_props} platform='dxtrade' />);
+        expect(screen.getByText(/IcBrandDxtrade/i).closest('a')).toHaveAttribute('href', 'https://dx.deriv.com');
+        expect(screen.getByText(/IcInstallationGoogle/i).closest('a')).toHaveAttribute(
+            'href',
+            'https://play.google.com/store/apps/details?id=com.deriv.dx'
+        );
+        expect(screen.getByText(/IcInstallationApple/i).closest('a')).toHaveAttribute(
+            'href',
+            'https://apps.apple.com/us/app/deriv-x/id1563337503'
+        );
+    });
```

Comment:
can we have  test cases with `active_index =1` so that demo account dashboard is covered?

Reply:
Nope... On it